### PR TITLE
Modified(Ai.cpp, Game.h, Main,cpp,Pch.h) { Comments translation; }

### PR DESCRIPTION
**Pull request by ntnt**
Further to english translation.

### CODE REVIEW

File: project/source/Pch.h [hunk 1]
```diff
@@ -14,11 +14,11 @@
 #define _WINSOCKAPI_
 
 //-----------------------------------------------------------------------------
-// warning o li�cie inicjalizacyjnej [struct C { C() : elem() {} int elem[10]; ]
+// initialization list compiler warning [struct C { C() : elem() {} int elem[10]; ]
 #pragma warning (disable: 4351)
 
 //-----------------------------------------------------------------------------
-// obs�uga wersji debug
+// debug version service
```
Comment (Tomash667) [inline project/source/Pch.h]: debug version handling

File: project/source/game/Ai.cpp [hunk 2]
```diff
@@ -60,35 +60,36 @@ cstring str_ai_idle[AIController::Idle_Max] = {
...
 enum MOVE_TYPE
 {
-	DontMove, // nie rusza si�
-	MoveAway, // odchodzi od punktu
-	MovePoint, // idzie w stron� punktu
-	KeepDistance, // stara si� zachowa� dystans 8-10 metr�w
-	KeepDistanceCheck, // jak wy�ej ale sprawdza czy z nowej pozycji mo�e trafi� wroga
-	RunAway // ucieka od punktu
+	DontMove,				
+	not moving
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: something went bad with comment here

File: project/source/game/Ai.cpp [hunk 3]
```diff
@@ -60,35 +60,36 @@ cstring str_ai_idle[AIController::Idle_Max] = {
...
+	MoveAway,				// moving from the point
+	MovePoint,				// moving to the point
+	KeepDistance,			// keep 8-10 meter distance
+	KeepDistanceCheck,		// keep 8-10 distance AND check if enemy is possible to hit
+	RunAway					// running to the point
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: running away from the point

File: project/source/game/Ai.cpp [hunk 4]
```diff
@@ -60,35 +60,36 @@ cstring str_ai_idle[AIController::Idle_Max] = {
...
 };
 
 enum RUN_TYPE
 {
-	Walk, // zawsze chodzenie
-	WalkIfNear, // chodzenie je�li cel jest blisko
-	Run, // zawsze bieg
-	WalkIfNear2
+	Walk,					// always walk
+	WalkIfNear,				// walk if target is nearby
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: you can add that near means 1,5m here :P

File: project/source/game/Ai.cpp [hunk 5]
```diff
@@ -60,35 +60,36 @@ cstring str_ai_idle[AIController::Idle_Max] = {
...
+	Run,					// always run
+	WalkIfNear2				// ???
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: like WalkIfNear but 3m distance

File: project/source/game/Ai.cpp [hunk 6]
```diff
@@ -702,29 +703,29 @@ void Game::UpdateAi(float dt)
 						{
 							if(IS_SET(u.data->flags2, F2_XAR))
 							{
-								// szukaj o�tarza
+								// search the altar
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: search for altar

File: project/source/game/Ai.cpp [hunk 7]
```diff
@@ -1249,7 +1250,7 @@ void Game::UpdateAi(float dt)
 												if(ai2.state == AIController::Idle
 													&& OR3_EQ(ai2.idle_action, AIController::Idle_None, AIController::Idle_Rot, AIController::Idle_Look))
 												{
-													// odwzajemnij patrzenie si�
+													// look at the person in response to his
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: .. in response to his looking

File: project/source/game/Ai.cpp [hunk 8]
```diff
@@ -1394,7 +1395,7 @@ void Game::UpdateAi(float dt)
 										}
 										else
 										{
-											// obr�� si� w strone u�ywalnego
+											// rotate towards user
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: rotate towards usable object

File: project/source/game/Ai.cpp [hunk 9]
```diff
@@ -1880,12 +1881,12 @@ void Game::UpdateAi(float dt)
 				break;
 
 			//===================================================================================================================
-			// id� do ostatnio znanego miejsca w kt�rym by� wr�g
+			// go last known enemy position
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: go *to* last...

File: project/source/game/Ai.cpp [hunk 10]
```diff
@@ -1900,7 +1901,7 @@ void Game::UpdateAi(float dt)
 							ai.alert_target = nullptr;
 						else
 						{
-							// kto� inny go zauwa�y�
+							// spotted by someone
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: someone else spotted enemy

File: project/source/game/Ai.cpp [hunk 11]
```diff
@@ -2122,18 +2123,18 @@ void Game::UpdateAi(float dt)
 						}
 						else if(!ai.escape_room)
 						{
-							// jeszcze nie ma ustalonego gdzie ucieka�
+							// escape location not known yet
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: escape location not *set* yet

File: project/source/game/Ai.cpp [hunk 12]
```diff
@@ -2641,15 +2642,15 @@ void Game::UpdateAi(float dt)
...
+				// door opening
 				if(ctx.doors && !IS_SET(u.data->flags, F_DONT_OPEN))
 				{
 					for(vector<Door*>::iterator it = ctx.doors->begin(), end = ctx.doors->end(); it != end; ++it)
 					{
 						Door& door = **it;
 						if(door.IsBlocking() && door.state == Door::Closed && door.locked == LOCK_NONE && Vec3::Distance(door.pos, u.pos) < 1.f)
 						{
-							// otw�rz magicznie drzwi :o
+							// open magic doors :o
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: when ai moves next to door they are automatically opened 🤣

File: project/source/game/Ai.cpp [hunk 13]
```diff
@@ -3012,7 +3013,7 @@ void Game::AI_Shout(LevelContext& ctx, AIController& ai)
 }
 
 //=================================================================================================
-// je�li target jest nullptr to atak nic nie zadaje (trenuje walk� na manekinie)
+// attack does not imply damage for nullptr target (dummy training)
```
Comment (Tomash667) [inline project/source/game/Ai.cpp]: when target is nullptr it deals no damage (dummy training)

File: project/source/game/Game.h [hunk 14]
```diff
@@ -490,9 +490,9 @@ class Game final : public Engine
 	void LeaveLevel(LevelContext& ctx, bool clear);
 	void UpdateContext(LevelContext& ctx, float dt);
 	bool IsAnyoneTalking() const;
-	// to by mog�o by� globalna funkcj�
+	// this could be a global function
 	void PlayHitSound(MATERIAL_TYPE mat_weapon, MATERIAL_TYPE mat_body, const Vec3& hitpoint, float range, bool dmg);
-	// wczytywanie
+	// reading
```
Comment (Tomash667) [inline project/source/game/Game.h]: loading

File: project/source/game/Game.h [hunk 15]
```diff
@@ -591,11 +591,11 @@ class Game final : public Engine
 	struct WarpData
 	{
 		Unit* u;
-		int where; // -1-z budynku, >=0-do budynku
+		int where; // <-1 - from the building,  >=0 - towards the building
```
Comment (Tomash667) [inline project/source/game/Game.h]: -1 get outside building, >=0 into building

File: project/source/game/Ai.cpp (hunk 13 above)
Comment (ntnt) [reply project/source/game/Ai.cpp]: I think my version sounds better better but ok

**Pull request merged**